tests/zfs-tests/tests/functional/ctime/Makefile.am

```diff
@@ -3,10 +3,11 @@ include $(top_srcdir)/config/Rules.am
 pkgdatadir = $(datadir)/@PACKAGE@/zfs-tests/tests/functional/ctime
 
 dist_pkgdata_SCRIPTS = \
+	ctime_001_pos.ksh \
 	cleanup.ksh \
 	setup.ksh
 
 pkgexecdir = $(datadir)/@PACKAGE@/zfs-tests/tests/functional/ctime
 
-pkgexec_PROGRAMS = ctime_001_pos
-ctime_001_pos_SOURCES = ctime_001_pos.c
+pkgexec_PROGRAMS = ctime
+ctime_001_pos_SOURCES = ctime.c
```

Comment:
Shouldn't this now be `ctime_SOURCES = ctime.c`?

Reply:
@behlendorf ..Done